Isabelle/document/root.tex~

```diff
@@ -0,0 +1,59 @@
+\documentclass[11pt,a4paper]{article}
```

Comment:
jEdit sometimes automatically generates `some.file~`!

Reply:
Removed the file, but let me check why Travis fails to build.

Reply:
Travis fails since it does not have pdflatex installed in the docker container. This is required to build the documentation and builds will fail entirely if this is not in place. Not sure how to resolve this. Options:

- We could exclude the document building from Isabelle/ROOT. Documentation building then needs to be invoked with the correct command line params locally (but we don't have any tests on Travis if that works).
- We could try to find a docker container that also has the `pdflatex` package installed.

Reply:
>* We could exclude the document building from Isabelle/ROOT. Documentation building then needs to be invoked with the correct command line params locally (but we don't have any tests on Travis if that works).

👍

Reply:
We can create two sessions in ROOT, so no need for special command line options. Builds are passing now :)